.pre-commit-config.yaml

```diff
@@ -1,10 +1,12 @@
 ---
 repos:
   - repo: "https://github.com/bufbuild/buf"
-    rev: "v1.47.2"
+    rev: "v1.54.0"
     hooks:
+      - id: "buf-generate"
```

Comment:
I think that in addition to checking the freshness of the generated file at commit time, we should have a documented command for imperatively generating the files, otherwise the process is a bit developer-hostile since you have to wait for a failure and then amend your commit and then re-commit